### DISCUSSION TITLE
Seed node?

### DISCUSSION THREAD

**Discussion by Centurion-Rome** [Comfy-Org/ComfyUI#995] [Q&A]
How can I enter a specific seed and text node and use its value (the same) for base, upscale and refiner?

**Comment by lord-lethris** [comment]
Like this:

Right click on the KSampler node to turn "Seed" into an input...

![image](https://github.com/comfyanonymous/ComfyUI/assets/1736720/5ed23151-63f1-46d4-b12e-2f7a5de54712)

You can then use a seed Node with fixed output OR the KSampler will take any INT input

![image](https://github.com/comfyanonymous/ComfyUI/assets/1736720/9fb7f172-27fb-41c7-b28d-0bd09b1461ab)

**Reply by ugintl** [reply]
I do not see any such option

**Reply by lord-lethris** [reply]
that's because they have since taken that feature away.

Instead any entry can be a "join".  Like this:

![image](https://github.com/user-attachments/assets/92a89060-e6a0-40cc-9959-389c17533e03)
![image](https://github.com/user-attachments/assets/1e3c899b-1773-4b08-987a-1963c2a7744b)
![image](https://github.com/user-attachments/assets/cd1ecb81-ccde-4e8e-b933-1aa956bdad23)
